### ISSUE TITLE
Removing playlist as first action after opening causes deadbeef crash

### ISSUE THREAD

**Issue by PainlessRob** [Alexey-Yakovenko/deadbeef#1224]
**System**: Arch Linux x86_64
**To reproduce**: Create a new playlist.  Play a song.  Close deadbeef.  Open deadbeef.  Remove the playlist you just created.  Deadbeef will crash.
**Trace from running deadbeef from CLI**:
Segmentation Fault
backtrace() returned 33 addresses
deadbeef() [0x403b01]
/usr/lib/libc.so.6(+0x335b0) [0x7f469378a5b0]
deadbeef() [0x40fbf8]
deadbeef() [0x40a179]
deadbeef() [0x40aa67]
deadbeef() [0x40b62b]
/usr/lib/deadbeef/ddb_gui_GTK2.so(draw_column_data+0xef) [0x7f469350cbdf]
/usr/lib/deadbeef/ddb_gui_GTK2.so(ddb_listview_list_render_row_foreground+0x158) [0x7f46934fbfd8]
/usr/lib/deadbeef/ddb_gui_GTK2.so(ddb_listview_list_render+0x544) [0x7f46934fce64]
/usr/lib/deadbeef/ddb_gui_GTK2.so(ddb_listview_list_expose_event+0x82) [0x7f46934fd3f2]
/usr/lib/libgtk-x11-2.0.so.0(+0x13190f) [0x7f4692f7890f]
/usr/lib/libgobject-2.0.so.0(g_closure_invoke+0x145) [0x7f46917e72f5]
/usr/lib/libgobject-2.0.so.0(+0x2202c) [0x7f46917f902c]
/usr/lib/libgobject-2.0.so.0(g_signal_emit_valist+0xae5) [0x7f4691801195]
/usr/lib/libgobject-2.0.so.0(g_signal_emit+0x8f) [0x7f46918018ef]
/usr/lib/libgtk-x11-2.0.so.0(+0x248bac) [0x7f469308fbac]
/usr/lib/libgtk-x11-2.0.so.0(gtk_main_do_event+0x552) [0x7f4692f77692]
/usr/lib/libgdk-x11-2.0.so.0(+0x4234f) [0x7f4692bd334f]
/usr/lib/libgdk-x11-2.0.so.0(+0x422f5) [0x7f4692bd32f5]
/usr/lib/libgdk-x11-2.0.so.0(+0x422f5) [0x7f4692bd32f5]
/usr/lib/libgdk-x11-2.0.so.0(+0x3ee28) [0x7f4692bcfe28]
/usr/lib/libgdk-x11-2.0.so.0(gdk_window_process_all_updates+0x118) [0x7f4692bd0758]
/usr/lib/libgdk-x11-2.0.so.0(+0x3f7b9) [0x7f4692bd07b9]
/usr/lib/libgdk-x11-2.0.so.0(+0x1e577) [0x7f4692baf577]
/usr/lib/libglib-2.0.so.0(g_main_context_dispatch+0x15d) [0x7f4690e1690d]
/usr/lib/libglib-2.0.so.0(+0x49ce0) [0x7f4690e16ce0]
/usr/lib/libglib-2.0.so.0(g_main_loop_run+0xc2) [0x7f4690e17002]
/usr/lib/libgtk-x11-2.0.so.0(gtk_main+0xb7) [0x7f4692f76467]
/usr/lib/deadbeef/ddb_gui_GTK2.so(gtkui_thread+0x74d) [0x7f46934e22ad]
/usr/lib/deadbeef/ddb_gui_GTK2.so(+0x2e4e9) [0x7f46934e24e9]
deadbeef() [0x403710]
/usr/lib/libc.so.6(__libc_start_main+0xf0) [0x7f4693777790]
deadbeef() [0x4039e9]

**Comment by cboxdoerfer** [comment]
Can't reproduce. Am I supposed to add files to the new playlist? Does it matter what playlist the song to be played belongs to? Do you have grouping enabled? Do you have album art enabled? Do you have resume playback after session enabled? Do you have listview appearance override enabled? What version of deadbeef do you use?

**Comment by PainlessRob** [comment]
Sincerest apologies for the lack of required details.

_Am I supposed to add files to the new playlist?_
- I added a folder to the new playlist.
- Played a song from the new playlist.
- Closed deadbeef.
- Opened deadbeef.
- Removed the playlist I had previously created (right-click on the playlist tab and select "Remove playlist").
Deadbeef crashes for me every time.

_Does it matter what playlist the song to be played belongs to?_
It was a song in the second of two the playlists (the same playlist I tried to remove).
I have just tried the same with three playlists, and got the same result.

_Do you have grouping enabled?_
I have no idea what grouping is, so probably not.

_Do you have album art enabled?_
No.

_Do you have resume playback after session enabled?_
I see no option to "resume playback after session", do you mean "resume previous session on startup"?  Either way, probably not, or no.

_Do you have listview appearance override enabled?_
Another option I'm completely unfamiliar with, so probably not.

_What version of deadbeef do you use?_
0.6.2

I'm using the Alsa output plugin.

Basically I just use deadbeef to play music.  I don't tend to make changes from the default, so most default options will be enabled.

**Comment by Alexey-Yakovenko** [comment]
Looks like this:
```
commit 05f3d65bb98cb7f7756ea6ff2632f7fcda7c9fbb
Author: Alexey Yakovenko <waker@users.sourceforge.net>
Date:   Fri Sep 12 21:24:16 2014 +0200

    gtkui: fix random crash after closing playlist using menu (bug #1193)
```

The bug probably won't happen if you use the official build from [here](http://deadbeef.sourceforge.net/download.html)

**Comment by Alexey-Yakovenko** [comment]
duplicate of bug #1095
